lib/rbUtil.js

```diff
@@ -173,6 +173,23 @@ rbUtil.parseETag = function(etag) {
     }
 };
 
+/**
+ * Copies forwarded headers from restbase to request.
+ * If the same header was already set it takes precedence over
+ * the forwarded header.
+ */
```

Comment:
One issue is that a) is not addressed currently. If we always forward cookies, then we might end up storing restricted content without also enforcing the same restrictions on storage access.